From 8b69ef02f67ade4bfd27547ed351aef810844159 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Fri, 12 Sep 2014 18:11:40 -0400
Subject: [powerpc] Bump COMMAND_LINE_SIZE to 2048

Message-id: <1410545655-205645-472-git-send-email-dzickus@redhat.com>
Patchwork-id: 94424
O-Subject: [RHEL7 PATCH 471/626] powerpc: Bump COMMAND_LINE_SIZE to 2048
Bugzilla: 1127366
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: David Gibson <dgibson@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1127366

commit a5980d064fe292584da73ddd180f4f140f9ab97d
Author: Anton Blanchard <anton@samba.org>
Date:   Mon Apr 14 21:54:52 2014 +1000

    powerpc: Bump COMMAND_LINE_SIZE to 2048

    I've had a report that the current limit is too small for
    an automated network based installer. Bump it.

    Signed-off-by: Anton Blanchard <anton@samba.org>
    Signed-off-by: Benjamin Herrenschmidt <benh@kernel.crashing.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/powerpc/include/uapi/asm/setup.h b/arch/powerpc/include/uapi/asm/setup.h
index 552df83..ae3fb68 100644
--- a/arch/powerpc/include/uapi/asm/setup.h
+++ b/arch/powerpc/include/uapi/asm/setup.h
@@ -1 +1,6 @@
-#include <asm-generic/setup.h>
+#ifndef _UAPI_ASM_POWERPC_SETUP_H
+#define _UAPI_ASM_POWERPC_SETUP_H
+
+#define COMMAND_LINE_SIZE 2048
+
+#endif /* _UAPI_ASM_POWERPC_SETUP_H */
-- 
1.7.1